docs/src/main/asciidoc/dev-services.adoc

```diff
@@ -60,84 +89,95 @@ N.B. if you opt in for this feature, Quarkus will not reset the state of the dat
 
-== RabbitMQ
+=== RabbitMQ
 
 The RabbitMQ Dev Service will be enabled when the `quarkus-messaging-rabbitmq` extension is present in your application, and
 the broker address has not been explicitly configured. More information can be found in the
 xref:rabbitmq-dev-services.adoc[RabbitMQ Dev Services Guide].
 
 include::{generated-dir}/config/quarkus-smallrye-reactivemessaging-rabbitmq-config-group-rabbit-mq-dev-services-build-time-config.adoc[opts=optional, leveloffset=+1]
 
-== Pulsar
+=== Pulsar
 
 The Pulsar Dev Service will be enabled when the `quarkus-messaging-pulsar` extension is present in your application, and
 the broker address has not been explicitly configured. More information can be found in the
 xref:pulsar-dev-services.adoc[Pulsar Dev Services Guide].
 
-== Redis
+=== Redis
 
 The Redis Dev Service will be enabled when the `quarkus-redis-client` extension is present in your application, and
 the server address has not been explicitly configured. More information can be found in the
 xref:redis-dev-services.adoc[Redis Dev Services Guide].
 
 include::{generated-dir}/config/quarkus-redis-config-group-client-dev-services-config.adoc[opts=optional, leveloffset=+1]
 
-== Vault
+=== Vault
 
 The Vault Dev Service will be enabled when the `quarkus-vault` extension is present in your application, and
 the server address has not been explicitly configured. More information can be found in the
 link:{vault-guide}#dev-services[Vault Guide].
 
-== Neo4j
-
-The Neo4j Dev Service will be enabled when the `quarkus-neo4j` extension is present in your application, and
-the server address has not been explicitly configured. More information can be found in the
-link:{neo4j-guide}#dev-services[Neo4j Guide].
-
-== Infinispan
+=== Infinispan
 
 The Infinispan Dev Service will be enabled when the `quarkus-infinispan-client` extension is present in your application, and
 the server address has not been explicitly configured. More information can be found in the
 xref:infinispan-dev-services.adoc[Infinispan Dev Services Guide].
 
 include::{generated-dir}/config/quarkus-infinispan-client-config-group-infinispan-client-build-time-config-dev-service-configuration.adoc[opts=optional, leveloffset=+1]
 
-== Elasticsearch
+=== Elasticsearch
 
 The Elasticsearch Dev Service will be enabled when one of the Elasticsearch based extensions (Elasticsearch client or Hibernate Search ORM Elasticsearch)
 is present in your application, and the server address has not been explicitly configured.
 More information can be found in the xref:elasticsearch-dev-services.adoc[Elasticsearch Dev Services Guide].
 
 include::{generated-dir}/config/quarkus-elasticsearch-devservices-elasticsearch-dev-services-build-time-config.adoc[opts=optional, leveloffset=+1]
+
+== Dev Services beyond the Quarkus Platform
+
+Many Quarkiverse extensions which are not in the Quarkus Platform also offer Dev Services.
```

Comment:
Would you consider DevServices outside of the Quarkiverse organization?
E.g. https://github.com/microcks/microcks-quarkus

Reply:
Definitely. We need to do a blog post clarifying all the terminology here, because it’s concentric rings crossed with Venn diagrams, with overlap and confusions. There's platform, and core, and quarkiverse, and GitHub, and sometimes they're related, and sometimes they're independent.  

But the TLDR is that Quarkiverse means the *whole* Quarkus ecosystem, not just ones living on the Quarkiverse GitHub org. The GitHub org makes releasing a bit easier, but it's not compulsory in any way. So Microcks is a good addition. :)

Reply:
I've added a second commit with the Microcks addition, thanks for the suggestion, @oleg-nenashev.